### PR TITLE
Add integration test for the Config sample

### PR DESCRIPTION
**Pull request by meltsufin**
Helps with #705.

### CODE REVIEW

File: spring-cloud-gcp-samples/spring-cloud-gcp-config-sample/src/test/java/ApplicationTests.java [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+@RunWith(SpringRunner.class)
+@SpringBootTest(classes = { Application.class }, webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+@AutoConfigureMockMvc
+public class ApplicationTests {
+	@Autowired
+	private MockMvc mvc;
+
+	@BeforeClass
+	public static void enableTests() {
+		assumeThat(System.getProperty("it.config")).isEqualTo("true");
```
Comment (ChengyuanZhao) [inline spring-cloud-gcp-samples/spring-cloud-gcp-config-sample/src/test/java/ApplicationTests.java]: Also add the -Dit.config=true in the travis.yaml file?

File: spring-cloud-gcp-samples/spring-cloud-gcp-config-sample/src/test/java/ApplicationTests.java [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.content;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+/**
+ * @author Mike Eltsufin
+ */
+@RunWith(SpringRunner.class)
+@SpringBootTest(classes = { Application.class }, webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+@AutoConfigureMockMvc
+public class ApplicationTests {
```
Comment (ChengyuanZhao) [inline spring-cloud-gcp-samples/spring-cloud-gcp-config-sample/src/test/java/ApplicationTests.java]: This class is missing a package statement

File: spring-cloud-gcp-samples/spring-cloud-gcp-config-sample/src/test/java/ApplicationTests.java (hunk 1 above)
Comment (meltsufin) [reply spring-cloud-gcp-samples/spring-cloud-gcp-config-sample/src/test/java/ApplicationTests.java]: It's already there.

File: spring-cloud-gcp-samples/spring-cloud-gcp-config-sample/src/test/java/ApplicationTests.java (hunk 2 above)
Comment (meltsufin) [reply spring-cloud-gcp-samples/spring-cloud-gcp-config-sample/src/test/java/ApplicationTests.java]: ah true!

**Pull request merged**